python/tests/test_topology.py

```diff
@@ -7917,3 +7917,69 @@ def test_is_isolated_bad(self):
                 tree.is_isolated("abc")
             with pytest.raises(TypeError):
                 tree.is_isolated(1.1)
+
+
+class TestExampleTrees:
+    """
+    Test the creation of hard-coded example tree (sequences)
+    """
+
+    def test_star_equivalent(self):
+        for extra_params in [{}]:  # , {'span':2.5}]: # Add after #930 is merged
+            for n in range(0, 6):
+                ts = tskit.Tree.create_star(n, **extra_params).tree_sequence
+                if n == 0:
```

Comment:
I think it'd be clearer to have a separate ``test_empty`` method, rather than embedding this into a loop.

Reply:
Sure